CMakeLists.txt

```diff
@@ -18,6 +18,13 @@ option(BUILD_DOCUMENTATION "Create HTML based API documentation (requires Doxyge
 option(RUN_LONG_UNIT_TESTS "Tests that takes more than 5 minutes to complete. No effect if BUILD_TESTING is OFF" OFF)
 option(BUILD_STORAGE_SAMPLES "Build sample application for Azure Storage clients" OFF)
 
+# Use the static runtime libraries when building statically for consistency with vcpkg's
+# arch-windows-static triplets:
+cmake_policy(SET CMP0091 NEW) # https://cmake.org/cmake/help/v3.15/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html
+if (NOT DEFINED CMAKE_MSVC_RUNTIME_LIBRARY)
```

Comment:
This was newly added in cmake 3.15 and our cmake currently requires 3.13 (with hopes to drop it lower). Using this is going in the other direction :p

I don't have a great alternative or solution, but maybe you can come up with something that works with an older version of cmake.